editor.planx.uk/src/pages/FlowEditor/ReadMePage/tests/ReadMePage.test.tsx

```diff
@@ -46,48 +47,52 @@ describe("Read Me Page component", () => {
     expect(screen.getByText("a summary")).toBeInTheDocument();
   });
 
-  it("displays an error if the service description is longer than 120 characters", async () => {
-    const { user } = setup(
-      <DndProvider backend={HTML5Backend}>
-        <ReadMePage {...defaultProps} />
-      </DndProvider>
-    );
+  it(
+    "displays an error if the service description is longer than 120 characters",
+    { timeout: 9000 },
```

Comment:
Lots of linting noise - but this line and `import { it } from "vitest";` (in order to support the timeout arg) are the only changes here !